index.js

```diff
@@ -2,4 +2,16 @@
 
 module.exports = {
   name: require('./package').name,
+
+  options: {
+    '@embroider/macros': {
+      setOwnConfig: {},
+    },
+  },
+
+  config(_emberEnv, config) {
+    // Pass `rootElement` config into a location where it can be looked up at run-time
+    this.options['@embroider/macros'].setOwnConfig.rootElement =
+      config.APP.rootElement;
+  },
```

Comment:
This approach was more-or-less borrowed from the `@embroider/macros` docs

https://github.com/embroider-build/embroider/tree/master/packages/macros#getownconfig-getconfig-and-getglobalconfig

But tweaked slightly to set the config from a location where we know what the consume app's `config/environment.js` file produced